README.md

```diff
@@ -45,15 +43,29 @@ See sample ansible playbook `site.yml.sample` to check how it fits together.
 
 Ansible Driven installation:
 
-* Step 1: Install Ansible >= 2.2
-* Step 2: Git the code:        `git clone https://github.com/Tendrl/tendrl-ansible.git`
-* Step 3: Set up the Ansible groups in inventory file:
-          `[ tendrl-server]       [ceph-servers]        [gluster-servers]`
-* Step 3: Create `site.yml` file based on `site.yml.sample` and make sure to
-  define `etcd_ip_address` and `tendrl_api_ip_address` to suit
-* Step 4: Run `# ansible-playbook site.yml`
-* Step 5: ssh into your Tendrl server as root and `cat /root/password` 
-* Step 6: log in to your tendrl server at http://ip.of.tendrl.server  with admin user and the password from step 5
+1) Install Ansible >= 2.2
+2) Get the code: `git clone https://github.com/Tendrl/tendrl-ansible.git`
+3) Create Ansible inventory file with groups for `tendrl-server`,
+   `ceph-servers` and/or `gluster-servers`. Here is an example of inventory
+   file for 4 node cluster with Gluster:
+
+```
+[gluster-servers]
+gl1.example.com
+gl2.example.com
+gl3.example.com
+gl4.example.com
+
+[tendrl-server]
+tendrl.example.com
+```
+
+4) Create `site.yml` file based on `site.yml.sample` and make sure to
+   define `etcd_ip_address` to suit
+5) Run `$ ansible-playbook -i inventory_file site.yml`
+6) Ssh into your Tendrl server as root and `cat /root/password` 
```

Comment:
I think this step is not necessary as the password is not random string but `adminuser` (https://github.com/Tendrl/api/pull/156/files).